lib/disc.rb

```diff
@@ -110,10 +119,14 @@ def disc_options
       end
 
       def queue
-        disc_options.fetch(:queue, 'default')
+        @queue || Disc.default_queue
```

Comment:
I don't think disc is supposed to be called multiple times. The last call wins, so the behavior shown above seems sane to me.

The supposed usage would be:
```ruby
class SomeJob
  include Disque::Job
  disc queue: "foo", retry: 300, ttl: 900
end
```